tsconfig.spec.json

```diff
@@ -5,7 +5,12 @@
     "types": [
       "jasmine",
       "node"
-    ]
+    ],
+    "paths": {
```

Comment:
Is this necessary? I would expect it to inherit `paths` from the base `tsconfig.json`.